src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -43,7 +38,6 @@ public AndroidBroker(CoreUIParent uiParent, ICoreLogger logger)
             _activity = uiParent?.Activity;
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _brokerHelper = new AndroidBrokerHelper(Application.Context, logger);
-            _readyForResponse = new SemaphoreSlim(0);
         }
 
         public bool CanInvokeBroker()
```

Comment:
Can you test what happens if you invoke the broker twice? I.e. call AcquireTokenInteractive and then call it imediatly again, without completing the broker flow. This should be a test for both iOS and Android